src/App.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import './App.scss';
+import Logic from './component/Logic/Logic';
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      logic: new Logic(),
+      running: false,
+      interval: 100
+    }
+
+    this.start = this.start.bind(this);
+    this.stop = this.stop.bind(this);
+    this.createBoard = this.createBoard.bind(this);
+    this.storeCell = this.storeCell.bind(this);
+  }
+
+  start() {
+    if(!this.state.running){
+      this.setState({
+        running: true,
+      }, () => {
+        this.intervalRef = setInterval(() => this.run(), this.state.interval);
```

Comment:
This is clever but not the most readable code, consider refactoring into a separate function with a descriptive name.